terminal.py

```diff
@@ -160,24 +162,48 @@ def call_stocks(self, _):
 
         return stocks_controller.menu()
 
+    def call_s(self, _):
+        """Process stocks command"""
+        from gamestonk_terminal.stocks import stocks_controller
```

Comment:
Instead of repeating code, can we just do a relay?

E.g.
```
def call_s(self, _):
   call_stocks(self, _)
```

Reply:
can't do it like that, but yes i relayed